collect_app/src/main/java/org/odk/collect/android/fragments/BarcodeWidgetScannerFragment.kt

```diff
@@ -8,7 +8,7 @@ class BarcodeWidgetScannerFragment : BarCodeScannerFragment() {
     }
 
     override fun shouldConfirm(): Boolean {
-        return true
+        return false
```

Comment:
We could entirely remove this feature, but I'd like to discuss whether we potentially want to offer an opt-in first.